src/data/faq_de.json

```diff
@@ -198,12 +198,12 @@
                         ]
                     },
                     {
-                        "title": "Wie kann ich ein Zertifikat aus der Corona-Warn-App entfernen?",
+                        "title": "Wie kann ich ein Zertifikat aus der Corona-Warn-App löschen?",
                         "anchor": "eu_dcc_remove",
                         "active": true,
                         "textblock": [
-                            "Mithilfe dieser Schritt-für-Schritt Anleitung können Sie Zertifikate aus der Corona-Warn-App entfernen. Bitte beachten Sie, dass die ersten drei Schritte für sowohl iOS als auch Android gelten, aber nur jeweils einer der letzten beiden Schritte befolgt werden muss, je nachdem welches Betriebssystem mit Ihrem übereinstimmt. Impfzertifikate, Testzertifikate und Genesenenzertifikate werden als 'Zertifikate' zusammengefasst, da der Ablauf bei allen identisch ist.",
-                            "<ul><li>Tippen Sie auf den Reiter 'Zertifikate'</li><li>Tippen Sie auf das Zertifikat, das Sie entfernen wollen</li><li>Scrollen Sie runter und tippen Sie auf die Kachel, die dem Zertifikat entspricht, das Sie löschen wollen</li><li>iOS: Tippen Sie unten auf dem Bildschirm auf 'Mehr' und dann auf 'Impfzertifikat löschen' bzw. 'Genesenen- oder Testzertifikat entfernen'</li><li>Android: Tippen Sie auf das drei-Punkte-Icon oben rechts auf dem Bildschirm, und danach auf 'Zertifikat entfernen' (bzw. auf 'Entfernen' für Version 2.9 und früher)</li></ul>"
+                            "Mithilfe dieser Schritt-für-Schritt Anleitung können Sie Zertifikate aus der Corona-Warn-App löschen. Bitte beachten Sie, dass die ersten drei Schritte für sowohl iOS als auch Android gelten, aber nur jeweils einer der letzten beiden Schritte befolgt werden muss, je nachdem welches Betriebssystem mit Ihrem übereinstimmt. Impfzertifikate, Testzertifikate und Genesenenzertifikate werden als 'Zertifikate' zusammengefasst, da der Ablauf bei allen identisch ist.<br>Nachdem Sie die nachfolgenden Schritte ausgeführt haben, wird das Zertifikat vorerst in den Papierkorb verschoben. Dort wird es dann nach 30 Tagen automatisch gelöscht, außer Sie stellen es von dort wieder her oder löschen es endgültig (Die Papierkorb-Funktion ist ab Version 2.13 verfügbar).",

```

Comment:
"... oder löschen es endgültig (Die Papierkorb-Funktion ..."

There should be a full-stop "." after "endgültig".

Reply:
Thanks Mike, I fixed the punctuation according to https://www.duden.de/sprachwissen/sprachratgeber/Klammern-in-Verbindung-mit-Komma-oder-Punkt.

Reply:
The German looks correct. The English is now wrong.